### DISCUSSION TITLE
NPM Package Transition

### DISCUSSION THREAD

**Discussion by Shivansh-Khunger** [Shivansh-Khunger/nitro#14] [Announcements]
This Project which was just standalone python scripts will be converted to a NPM package. 
This was in the works from the start of the project. 

The current state of the project would be preserved under a directory **deprecated** and the code would be updated accordingly if anyone wants to use version (idk why? 🥲). 

You can view the timeline [here](https://www.figma.com/board/S7WTGmXAAq2bA1iehqg1M7/Node-Nitro?node-id=0%3A1&t=xsDMiH7seKqGpEEs-1) .

**Comment by banyalvansh** [comment]
Great Decision 👍
